src/core/server/server.ts

```diff
@@ -128,7 +130,10 @@ export class Server {
 
   public async preboot() {
     this.log.debug('prebooting server');
-    const prebootTransaction = apm.startTransaction('server_preboot', 'kibana_platform');
```

Comment:
The underscore change is not critical, but I wanted to be consistent across all events (I know it's not mandatory, but I think it would be nice to have a convention).

Reply:
> I know it's not mandatory, but I think it would be nice to have a convention

What is the benefit of having one? These values are tracked by the code owners only.
How we are going to enforce this convention?  I would avoid introducing it without an enforcing mechanism because sooner or later there will be deviations in the codebase.

Reply:
My reasoning was:
(a) The dash syntax is the chosen convention APM uses internally.
(b) This data is exposed as is on the APM drowdowns and pages - it's going to be much more convenient to navigate if it's uniform.
(c) Once we stream data into a central location, which we are working on, more people will start looking and analyzing data other than code owners (devs, SREs and in the future *customers*). Maybe we will even have dedicated data analyst some day that's not too far away.

Like I said, I don't think it's crititcal. but since people often copy-paste these code bits, I thought it would be nice to standardize this. But I can also revert this if you prefer not to 👍🏻 .